frontend/add-client/demographic-information.component.tsx

```diff
@@ -219,7 +275,7 @@ export default function DemographicInformation(props: StepComponentProps) {
             )}
             <div>
               <label>
-                <span>Pay interval</span>
+                <span>Pay interval frequency</span>
```

Comment:
Doesn't `interval` already refer to how frequent something happens? So `interval frequency` is redundant?